### ISSUE TITLE
Typescript Linter

### ISSUE THREAD

**Issue by BentleyDavis** [SettleIt/ReasonRoot#19]
Add in a Typescript linter for code quality. I have not settled on one yet. Suggestions welcome.

**Comment by EdgarOPG** [comment]
I have this, please check it, what do you think about it?
https://palantir.github.io/tslint/

**Comment by BentleyDavis** [comment]
Yes, specifically Eslint with Airbnb config or tslint for typescript .

**Comment by BentleyDavis** [comment]
On second thought, lets just use the base set of rules first. We can add configs later.
